paywall/src/data-iframe/postOffice.js

```diff
@@ -70,6 +70,23 @@ export default function postOffice(window, requiredConfirmations) {
     },
   }
 
+  const lockHandler = cachedData => {
+    actions.locks(cachedData.locks)
+    const unlockedLocks = Object.values(cachedData.locks)
```

Comment:
we should consider renaming this to something that indicated that returns a collection of lockAddresses

Reply:
#3643 addresses this feedback